src/secrets/components/EditSecretOverlay.tsx

```diff
@@ -0,0 +1,40 @@
+// Libraries
+import React, {FC} from 'react'
+import {useHistory} from 'react-router-dom'
+import {useSelector} from 'react-redux'
+
+// Components
+import {Overlay} from '@influxdata/clockface'
+import EditSecretForm from 'src/secrets/components/EditSecretForm'
+
+// Types
+import ErrorBoundary from 'src/shared/components/ErrorBoundary'
+import GetResources from 'src/resources/components/GetResources'
+import {ResourceType} from 'src/types'
+import {getOrg} from 'src/organizations/selectors'
+
+const EditSecretOverlay: FC = () => {
+  const history = useHistory()
+  const orgId = useSelector(getOrg)
```

Comment:
I know I wrote this.

```suggestion
  const orgId = useSelector(getOrg)?.id
```